functions/fish_title.fish

```diff
@@ -2,11 +2,10 @@ function fish_title \
     --description "Set title to current folder and shell name" \
     --argument-names last_command
 
-    set --local basename (string replace --regex '^.*/' '' -- $PWD)
-    set --local current_folder (_pure_parse_directory)
```

Comment:
Thanks for the introducing me to `fish_prompt_pwd_dir_length`!
Would be better to keep `_pure_parse_directory` and edit its definition to impact both the window's title and the prompt. So, we have a consistent behaviour.

Reply:
Ideally we could make the directory length configurable for the prompt and the title based on a different variable.

My personal use case is to have the full directory structure in my prompt but a shorten one for the title. As a user of [Yakuake](https://apps.kde.org/yakuake/) having long title is annoying. Would you be ok having two feature variable ?

Reply:
I'm ok having two feature variables, I'm following the [pattern](https://github.com/pure-fish/pure/blob/master/CONTRIBUTING.md#features-variables) `pure_<verb>_<feature>`:

**Shorten Length in window's title**
```fish
pure_shorten_window_title_current_directory_length
```

**Shorten Lenght in prompt**
```fish
pure_shorten_prompt_current_directory_length
```